schema.go

```diff
@@ -483,8 +483,10 @@ func (api *API) RegisterModel(model Model, opts ...ModelOpts) (name string, sche
 				fieldName = f.Name
 			}
 
+			validationTags := strings.Split(f.Tag.Get("validate"), ",")
+
 			tempOpts := []ModelOpts{}
-			if len(jsonTags) > 1 && jsonTags[1] == "omitempty" {
+			if slices.Contains(validationTags, "omitempty") {
```

Comment:
suggestion: can't we just check the string returned by `f.Tag.Get()` if it contains "omitempty"? I can't see another usage of `validationTags`, so it shouldn't hurt?